src/index.js

```diff
@@ -67,6 +57,36 @@ const useKeyboardShortcuts = (
       )}. Found event: "${eventType}".`
     )
 
+  const shortcutHasPrioroty = (inputShortcut, event) => {
+    if (shortcuts.length === 1) return true
+    if (isSingleKeyShortcut(inputShortcut) && isSingleKeyEvent(event))
+      return true
+
+    return !shortcuts.find(
+      shortcut =>
+        allComboKeysPressed(shortcut.keys, event) &&
+        shortcut.keys.length > inputShortcut.keys.length
+    )
+  }
+
+  const generateFunction = (shortcut, event) => {
```

Comment:
Does this generate a function? It does look like it rather calls `callCallback` or similar?